New docs area for URL semantics, relocate some HTTP pages here (#35202)
mdn/content: files/en-us/web/uri/authority/index.md

---
title: URI authority
slug: Web/URI/Authority
page-type: guide
spec-urls: https://www.rfc-editor.org/rfc/rfc3986.html#section-3.1
---

{{QuickLinksWithSubpages("/en-US/docs/Web/URI")}}

> The **authority** of a URI is the part of the URI that comes after the scheme and before the path. It consists of three parts: user information, host, and port.

Reviewer: ```suggestion
The **authority** of a URI is the part that comes after the scheme and before the path. It may have up to three parts: user information, host, and port.
```

Reviewer: and link to the scheme doc.

## Syntax

```url
host
host:port
user@host
user@host:port
```

- host
  - : The _host_ is usually the domain name or IP address of the server hosting the resource. The domain name is resolved to an IP address using the {{glossary("DNS", "Domain Name System")}}.

> - port

Reviewer: ```suggestion
- port {{optional_inline}}
```

  - : The _port_ is a number that indicates the port on which the server is listening for requests. It is optional and defaults to 80 for HTTP and 443 for HTTPS. Other schemes may define their own defaults or make it mandatory.

> - user

Reviewer: ```suggestion
- user {{optional_inline}}
```
